slides/3-foundations-of-javascript.html

```diff
@@ -458,6 +557,26 @@ <h2><i class="fab fa-js"></i> Foundations of JavaScript</h2>
                         </textarea>
                     </section>
 
+                    <section data-markdown style="font-size: 0.75em;">
+                        <textarea data-template>
+                            ## <i class="fab fa-js"></i> String Operators
```

Comment:
Moved this slide here, makes more sense to me around binary operators than where it was before.